scanomatic/qc/compilation_results.py

```diff
@@ -1,6 +1,7 @@
 import glob
 import os
 import re
+from typing import Sequence
```

Comment:
shouldn't this be `collections.abc`?

Reply:
yes must have automatically imported from typing.